### ISSUE TITLE
Exception on Linux installation (4.5)

### ISSUE THREAD

**Issue by Manuel-Innovapps** [iAlexeyProkhorov/qBoSlider#12]
Nop 4.5
Linux AppService, hosted in Azure.
The plugin uses windows like file paths in Linux, which causes the following exception.

```
System.IO.IOException: Invalid argument : '/home/site/wwwroot/Plugins/Widgets.qBoSlider\Content\localization.en-US.xml'
   at Interop.ThrowExceptionForIoErrno(ErrorInfo errorInfo, String path, Boolean isDirectory, Func`2 errorRewriter)
   at Interop.CheckIo(Error error, String path, Boolean isDirectory, Func`2 errorRewriter)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String path, OpenFlags flags, Int32 mode)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
   at System.IO.Strategies.FileStreamHelpers.ChooseStrategy(FileStream fileStream, String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, Int64 preallocationSize)
   at System.IO.File.ReadAllBytes(String path)
   at Nop.Plugin.Widgets.qBoSlider.BaseBaroquePlugin.InstallLocalization()
   at Nop.Plugin.Widgets.qBoSlider.BaseBaroquePlugin.InstallAsync()
   at Nop.Plugin.Widgets.qBoSlider.qBoSliderPlugin.InstallAsync()
   at Nop.Services.Plugins.PluginService.InstallPluginsAsync() in /opt/atlassian/pipelines/agent/build/Libraries/Nop.Services/Plugins/PluginService.cs:line 502
```

**Comment by Manuel-Innovapps** [comment]
```
protected string GenerateLocalizationXmlFilePathByCulture(string culture = "en-US")
        {
            string fileName = string.Format("localization.{0}.xml", culture);
            string contentDirectoryPath = $"{_originalAssemblyFile.DirectoryName}{Path.DirectorySeparatorChar}Content{Path.DirectorySeparatorChar}";

            return $"{contentDirectoryPath}{fileName}";
        }
```

This should fix the issue

**Comment by iAlexeyProkhorov** [comment]
Hi! Thank you for your reply and fix!
I'll add fix into plugin 😉👍

**Comment by iAlexeyProkhorov** [comment]
Hi! Sorry for delay.
This issue was fixed in qBoSlider for nopCommerce 4.40, 4.50, 4.60
